docs/source/general/importing_users.rst

```diff
@@ -0,0 +1,44 @@
+===============
+Importing users
+===============
+
+This feature can be used for importing users from a csv file. There are many features included in it such as:
+
+* Importing users in batches: all of the users of a particular csv file would be stored in batches and can be retrieved/ deleted easily using the batch functions.
+* Set an expiration date: Expiration date can be set for a batch after which the users would not able to authenticate to the RADIUS Server.
+* Autogenerate usernames and passwords: The usernames and passwords are automatically generated if they aren't provided in the csv file. Usernames are generated from the email address whereas passwords are generated randomly where the length of passwords can be customized.
+* Passwords are accepted in both cleartext and hash formats from the CSV.
+
+We've defined management commands which would help us in acheiving these.
+
+``batch_add_users``
+-------------------
+
+This command imports users from a csv file. Usage is as shown below.
+
+.. code-block:: shell
+
+    ./manage.py batch_add_users --file <filepath> --expiration <expiration_date> --password-length <password_length>
+
+Note that the expiration and password-length are optional parameters which default to never and 8 respectively.
+
+It is important to take care of the following when importing users from the CSV.
```

Comment:
what about using subheadings for each point instead of a list? Wouldn't it be more readable?